python/repositories.bzl

```diff
@@ -278,6 +292,25 @@ python_repository = repository_rule(
     _python_repository_impl,
     doc = "Fetches the external tools needed for the Python toolchain.",
     attrs = {
+        "coverage_tool": attr.label(
+            # Mirrors the definition at
+            # https://github.com/bazelbuild/bazel/blob/master/src/main/starlark/builtins_bzl/common/python/py_runtime_rule.bzl
+            allow_files = False,
+            doc = """
```

Comment:
Lets also mention the py_runtime.coverage_tool docs and refer the reader there for more information. It'll be easy for these two attribute docs to get out of sync.

Reply:
Done.